File: src/sum-trig.c
#include <stdio.h>
#include <math.h>

int main(int argc, char *argv[]) {
  int ndat = 1000*1000*1000;
  double sum = 0.0;
  double dx = 0.0001;
  for (int i=0; i<ndat; ++i) {
    sum += cos(i*dx)*dx;
  }

  printf ("sum = %f\n", sum);
}

File: src/sum-trig.py
import math

ndat = 1000*1000*1000
dx = 0.0001
sum = 0.0

i = 0
while i < ndat:
    sum += math.cos(i*dx)*dx
    i += 1
    
print('sum = %f' % sum)

# (+ (* 60 5) 17) = 317 
